src/test/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStepTest.java

```diff
@@ -207,7 +207,7 @@ public void getDisplayName_ReturnsFunctionName() {
         String displayName = descriptor.getDisplayName();
 
         // then
-        assertThat(displayName).isEqualTo("office365ConnectorSend");
+        assertThat(displayName).isNotEqualTo("office365ConnectorSend");
```

Comment:
poor assertion

Reply:
Thx, I've changed a little bit this test case - please check.